packages/ember/tests/helpers/helper_registration_test.js

```diff
@@ -69,7 +61,6 @@ QUnit.test('Unbound dashed helpers registered on the container can be late-invok
   });
 
   equal(jQuery('#wrapper').text(), 'BORF YES', 'The helper was invoked from the container');
-  ok(!helpers['x-borf'], 'Container-registered helper doesn\'t wind up on global helpers hash');
```

Comment:
`_registerHelper` is private API and things should come out of the container. This does not have a complimentary thing in Glimmer.